src/utils/getPrinterLanguage.ts

```diff
@@ -0,0 +1,17 @@
+import { PRINTER_LANGUAGE } from '../constants';
+import type { PrinterLanguage } from '../types';
+
+/**
+ * @param language the language code
+ * @returns the language code of the printer depeding ot the passed language param, defaults to EPOS2_LANG_EN
+ */
+export function getPrinterLanguage(language: PrinterLanguage): number {
```

Comment:
Could you please describe this util in docs?

Reply:
This util function is used internally and I don't think it needs to be in the docs for the package usage. It might be used by contributors, but for them we have JSDoc.